### PR TITLE
Use -d to specify number of decimals in output.

### PR DESCRIPTION
**Pull request by sgh**
#807 I have done the necessary changes. I do however not know how to sufficiently test them. In addition to that I was hoping that the tools we more similar. The way things are done in `cct` seem better that the way thins are done in `proj` and `cs2cs` and I was not really up to migrating all of them to the cct-way.

### CODE REVIEW

**Review by kbevers — CHANGES_REQUESTED** [review]
I think this looks all right. There's a few things missing documentation wise:

1) Add `-d` to the list of available flags in `proj.c` (line 53)
2) Add `-d` and `-D` to the list of available flags in `cs2cs` (line 56)
3) Add `-d` option to `cct`, `cs2cs` and `proj` doc-files
4) Add `-D` option `cs2cs` doc file.

Regarding testing, that is a bit complicated at the moment. The `proj` and `cs2cs` apps can be tested by using the test scripts in `nad/`. See https://github.com/OSGeo/proj.4/pull/1106/files for an example. A similar thing could be done for `cct` but I think that should be avoided and instead we need to make a proper test setup of the CLI apps. Hopefully we will get round to doing that soon.

**Comment by sgh** [comment]
Yeah - I did see the autogenerate comment ....

File: src/cs2cs.c [hunk 1]
```diff
@@ -293,10 +294,15 @@ int main(int argc, char **argv)
...
                 reverseout = 1;
                 continue;
-              case 'd': /* set debug level */
+              case 'D': /* set debug level */
                 if (--argc <= 0) goto noargument;
                 pj_ctx_set_debug( pj_get_default_ctx(), atoi(*++argv));
                 continue;
+              case 'd':
+                if (--argc <= 0) goto noargument;
+                snprintf(oform_buffer, sizeof(oform_buffer), "%%.%df", atoi(*++argv));
```
Comment (kbevers) [inline src/cs2cs.c]: The use of `snprintf` is what is causing Travis to trip up. Unfortunately `snprintf` is not C89  which is a requirement at the moment. While a bit more cumbersome it should be possible to do this with `sprintf` instead.

Comment (sgh) [reply src/cs2cs.c]: Oh ... that was the reason :) thanks.

**Comment by kbevers** [comment]
Thanks for adding this!

**Comment by sgh** [comment]
Thank you for the feedback ping-pong.

**Pull request merged**